src/main/java/uk/gov/hmcts/reform/idam/web/strategic/PolicyService.java

```diff
@@ -137,7 +158,7 @@ private boolean checkNoActionsBlockingUser(ResponseEntity<EvaluatePoliciesRespon
             return null;
         }
         final String[] splitArray = StringUtils.split(ipAddress, ",");
-        return Arrays.asList(splitArray).stream()
+        return asList(splitArray).stream()
```

Comment:
It would be better to use Arrays.stream(splitArray) here, to avoid creating a list.